spec/lib/dor/item_spec.rb

```diff
@@ -51,14 +50,12 @@
   end
 
   it 'should add the workflow for a collection' do
-    expect(Dor).to receive(:find).with(@druid).and_return(@dor_object).exactly(1).times
-    expect(@dor_object).to receive(:create_workflow).with(Dor::Config.release.workflow_name).exactly(1).times
+    expect(Dor::Config.workflow.client).to receive(:create_workflow).exactly(1).times
```

Comment:
one other thing:  might be nice to have some expectations on the parameter values (e.g. `expect(Dor::Config.workflow.client).to receive(:create_workflow).with(...)`)

Reply:
added expectations of parameters passed